tobac/themes/tint/objects.py

```diff
@@ -31,17 +31,20 @@ def check_isolation(raw, filtered, grid_size, params):
                                       params['ISO_THRESH'])
     nobj_iso = np.max(iso_filtered)
     iso = np.empty(nobj, dtype='bool')
+    neighbors = np.ones(nobj)*(-9999)
```

Comment:
Why is this set to -9999? I'm not that familiar with the tint code yet, so if there's something obvious, let me know. Trying to figure out missing data vs. 0 neighbors; is this better as 0?